recipes/libcusolvermp/meta.yaml

```diff
@@ -0,0 +1,118 @@
+{% set version = "0.5.1.690" %}
+{% set libcal_version = "0.4.4.50" %}
+{% set lib_version = version.split(".")[0]+"."+version.split(".")[1]+"."+version.split(".")[2]+".0" %}
+{% set platform = "linux-x86_64" %}  # [linux64]
+{% set platform = "linux-sbsa" %}    # [aarch64]
+{% set extension = "tar.xz" %}
+
+{% set cuda_major = environ.get("cuda_compiler_version", "12.6").split(".")[0] %}
+
+package:
+  name: libcusolvermp
+  version: {{ version }}
+
+source:
+  url: https://developer.download.nvidia.com/compute/cusolvermp/redist/libcusolvermp/{{ platform }}/libcusolvermp-{{ platform }}-{{ version }}_cuda{{ cuda_major }}-archive.{{ extension }}
```

Comment:
Please add `conda-forge.yml` with the following contents to the libcusolvermp directory:

```yaml
os_version:
  linux_64: cos7
  linux_aarch64: cos7
  linux_ppc64le: cos7
```

Because this is a redist we want the os version of the container to have the oldest supported glibc version. In this case glibc 2.17 (unless that's not the case?)

Reply:
Will do.